### ISSUE TITLE
Build error in IOS when Hermes is enabled

### ISSUE THREAD

**Issue by a-chess** [ammarahm-ed/react-native-admob-native-ads#212]
When I build a example project with hermes enabled, I get a build error in IOS.

**Podfile**
```rb
  use_react_native!(
    :path => config[:reactNativePath],
    :hermes_enabled => true
  )
```

**XCode**
<img src="https://user-images.githubusercontent.com/7105360/132637072-1457b06d-78c2-455d-883e-85043e82d2eb.png" width="500">

I'm not sure if this is the right way to do it, but I made the following modifications and confirmed that it works correctly.
https://github.com/a-chess/react-native-admob-native-ads/pull/1/files

Please let me know the right solution for me.

Thank you.

**Comment by ammarahm-ed** [comment]
@wjaykim Did you test with hermes enabled when you made those changes?

**Comment by wjaykim** [comment]
No, I didn't.
But I know the solution. I made change to header file import from `#import <React/xx.h>` to `@React`. I think we have to revert these changes.

**Comment by doranteseduardo** [comment]
I got those errors fixed by upgrading to RN 0.65.0

**Comment by zoobibackups** [comment]
> 6.9.0.1

after donwgrading I have these issues 

Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADAdLoader


Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADMultipleAdsAdLoaderOptions


Showing Recent Messages
Undefined symbol: _OBJC_METACLASS_$_GADNativeAdView



Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADCustomEventExtras



Showing Recent Messages
Undefined symbol: _GADSimulatorID



Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADCustomEventExtras

Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADNativeAdMediaAdLoaderOptions

Showing Recent Messages
Undefined symbol: _OBJC_CLASS_$_GADNativeAdViewAdOptions
